app/controllers/registrations_controller.rb

```diff
@@ -28,13 +28,17 @@ def create
     end
   end
 
+  def registrations_closed
+    render "registrations/registrations_closed"
+  end
+
   private
 
   def check_registrations_open_or_valid_invite!
     return true if AppConfig.settings.enable_registrations? || invite.try(:can_be_used?)
 
     flash[:error] = params[:invite] ? t("registrations.invalid_invite") : t("registrations.closed")
```

Comment:
Actually the flash message feels a bit redundant now, I'd just drop it for the non-invite case at least.

For the invalid invite case, I wonder if we should embed the wording on the page rather than doing a flash message which might be missed and confusing to show on the new page, but I'd say we can streamline that case in a follow up in order to move this forward.

```suggestion
     flash[:error] = t("registrations.invalid_invite") if params[:invite]
```

Reply:
Done ;)

Reply:
Forgot to push? :D

Reply:
Oh sorry, I pushed to my fork instead of Alebor's one.

Reply:
Looks like it's breaking some specs :)

Reply:
Sorry I messed up with remotes a bit and erased my previous work by pushing your changes. I was able to restore it so everything should be good now.